site/content/en/docs/contrib/external-packages.en.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Managing External Packages"
+linkTitle: "Managing External Packages"
+---
+
+## Managing External Packages
+
+### Chocolatey
+* It is a software management automation for Windows that wraps installers, executables, zips, and scripts into compiled packages.
+* View Minikube package on Chocolatey [here](https://community.chocolatey.org/packages/Minikube/).
```

Comment:
try to avoid view here verbage.. just describe what is at the link..
```
[Install the latest Minikube]
Get the [current status] of the Minikube chocolatey package
```

Then for the last link I'd separate that into another sentence.  Something about the Minikube chocolatey package is on github..

Reply:
Agreed. Please review and resolve as per the latest commit. Thank you.